neutron/tests/unit/agent/test_securitygroups_rpc.py

```diff
@@ -1111,6 +1111,7 @@ class BaseSecurityGroupAgentRpcTestCase(base.BaseTestCase):
     def setUp(self, defer_refresh_firewall=False):
         super(BaseSecurityGroupAgentRpcTestCase, self).setUp()
         set_firewall_driver(FIREWALL_NOOP_DRIVER)
+        ipv6._IS_IPV6_ENABLED = True
```

Comment:
It would be nice to have a test for the IPv6 disabled path to validate your new code, but it isn't critical.